### ISSUE TITLE
Where does the code reflect that "the global loss is weighted and accumulated by the loss functions of each branch"?

### ISSUE THREAD

**Issue by wxyWendy** [4uiiurz1/pytorch-nested-unet#47]
Where does the code reflect that "the global loss is weighted and accumulated by the loss functions of each branch"?

**Comment by wxyWendy** [comment]
I get it.
